source/_integrations/androidtv_remote.markdown

```diff
@@ -0,0 +1,382 @@
+---
+title: Android TV Remote
+description: Instructions on how to integrate Android TV remotes into Home Assistant.
+ha_category:
+  - Remote
+ha_release: 2023.4
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@tronikos'
+ha_quality_scale: platinum
+ha_domain: androidtv_remote
+ha_zeroconf: true
+ha_platforms:
+  - diagnostics
+  - remote
+ha_integration_type: device
+---
+
+The `androidtv_remote` platform allows you to control an Android TV device by sending [commands](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt) and launching apps.
```

Comment:
It isn't a platform?

```suggestion
The Android TV Remote integration allows you to control an Android TV device by sending [commands](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt) and launching apps.
```

Reply:
You are right. I copied this from https://www.home-assistant.io/integrations/harmony/ Fixing that one too in https://github.com/home-assistant/home-assistant.io/pull/26750